ttyplot.1

```diff
@@ -83,6 +83,13 @@ Print the current version and exit.
 .It Fl h
 Print this help message and exit.
 .El
+.Sh KEY BINDINGS
+When reading data from a pipe, ttyplot accepts the following commands typed at the terminal:
```

Comment:
Could you make "ttyplot" bold as used in other places, e.g. in the authors section, using `.Nm` (probably)?

Reply:
`.Nm` works. :smile: